<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnimationStripBuilder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon.png</normaloff>:/resources/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(59, 59, 59);
color:rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFirstFrame">
       <property name="text">
        <string>First frame</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditFirstFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(36, 36, 36);
border:1px solid white;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelResult">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(36, 36, 36);
border:1px solid white;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelNRows">
       <property name="text">
        <string>Rows number</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelNColumns">
       <property name="text">
        <string>Columns number</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QPushButton" name="pushButtonCreate">
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
	background-color:rgb(96, 96, 96);
}</string>
       </property>
       <property name="text">
        <string>Create!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="toolButtonSetResult">
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	background-color:rgb(96, 96, 96);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButtonSetFirstFrame">
       <property name="styleSheet">
        <string notr="true">QToolButton:hover {
	background-color:rgb(96, 96, 96);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonRowsCols">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Left -&gt; Right</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonColsRows">
         <property name="text">
          <string>Top -&gt; Downward</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditNRows">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(36, 36, 36);
border:1px solid white;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditNColumns">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(36, 36, 36);
border:1px solid white;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
